juniper_official/routing/check-te-rsvp-interface-errors.rule

```diff
@@ -235,6 +247,18 @@ healthbot {
                         next;
                     }
                 }
+                term no-bad-checksum {
```

Comment:
what if field is not present

Reply:
Then due to zero-suppression in the rule, it will match 0 and go to default condition of green.

Reply:
Added "when exists" for all green conditions.